src/Adapters/BlobAdapter.ts

```diff
@@ -438,7 +447,7 @@ export default class BlobAdapter implements IBlobAdapter {
                             .map((mapping) => mapping.allowedHeaders)
                             .reduce(
                                 (acc: boolean, headerGroup) =>
-                                    acc && headerGroup[0].includes(header),
+                                    acc && headerGroup?.includes(header),
```

Comment:
it's intentional that we now search the entire array instead of just the first entry?

Reply:
I changed it since I wanted to get this loweCase comparsion.